tests/microos/efiboot_check.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Basic smoke test to verify SUSE/openSUSE efiboot
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base qw(opensusebasetest);
+use testapi;
+use transactional qw(trup_call check_reboot_changes);
+use version_utils qw(is_sle_micro);
+use utils qw(assert_secureboot_status);
+
+sub run {
+    select_console('root-console');
+
+    trup_call('bootloader');
+    check_reboot_changes;
+
+    my $efiboot = script_output('efibootmgr -v', proceed_on_failure => 1);
+
+    unless ($efiboot == /^Boot\w{4}\*\s(opensuse|sle)-secureboot/) {
+        die 'System has booted from unexpected efi boot record!';
+    }
+
+    validate_script_output('mokutil --sb-state', sub { m/secureboot enabled/i });
```

Comment:
Do you think it makes sense to introduce a `SECURE_BOOT` variable that allows this test module to check for the efi boot, but not secure boot? Something like e.g.

```suggestion
    validate_script_output('mokutil --sb-state', sub { m/secureboot enabled/i }) unless (get_var("SECURE_BOOT", "1") == 0);
```

Reply:
would make sense for https://openqa.opensuse.org/tests/2834312#step/efiboot_check/27, but sure what is expected there.
@ggardet any idea?

Reply:
Not sure what is the default for self-install, but IMO, SecureBoot should be enabled by default.

Reply:
for sle-micro it is enabled, let me try to make a run for leap-micro 5.3 on x86_64

Reply:
https://openqa.opensuse.org/tests/2866763#step/efiboot_check/26

Reply:
Looks like you found a bug then. ;)

Reply:
> Looks like you found a bug then. ;)

Yay! :smile:  Going to report it then. Thanks

Reply:
https://bugzilla.opensuse.org/show_bug.cgi?id=1205299